one-time-password.cabal

```diff
@@ -85,3 +85,42 @@ test-suite tests
     , text
     , text-display
     , torsor
+
+executable cli
```

Comment:
Could you please change the name to the name of the package? Otherwise we get something like:

```
Copying 'cli' to '/home/hecate/.cabal/bin/cli'
```